src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -22,10 +23,12 @@ public class PatchMapDrawer : Control
     [Export]
     public Color ConnectionColour = new(1.0f, 1.0f, 1.0f, 1.0f);
 
+    [Export]
+    public ShaderMaterial MonochromeMaterial = null!;
```

Comment:
Can't Control modulation be used to make things monochrome? That's how it's done in the editor, I think.

Reply:
I don't really get what you mean by that. In the editor everything is made grayscale by disabling buttons.

Reply:
Yes. So I'm asking *why* you use a custom ShaderMaterial here instead of adding the module to gray option. This is for example done on the organelle selection buttons, we don't have separate icons for their selected or unselected state.

Reply:
Because modulation only darkens the tiles. I wanted to go for a monochrome effect. I guess this is personal taste, but I like the monochrome one more.

![image](https://user-images.githubusercontent.com/30809803/161396148-808f76d5-aa9c-493d-ab4f-23e950b212b2.png)
![image](https://user-images.githubusercontent.com/30809803/161396211-bb8e465d-f863-4443-bef5-f27a43d4b3b4.png)

Reply:
Hang on but then how do the selected organelle icons swap colours?

Reply:
The background gets light by "pressing" the button and gets gray by disabling it.
The Icon and text get gray by modulating down the white text to a grayer color.

Reply:
Okay, fine, I guess there's no other way to do this than to use a custom shader then.